lib/phlex/rails/helpers.rb

```diff
@@ -1149,8 +1149,25 @@ module SubmitTag
 	module T
 		extend Phlex::Rails::HelperMacros
 
-		# @!method t(...)
-		define_value_helper :t
+		def self.included(base)
+			base.extend(ClassMethods)
+		end
+
+		module ClassMethods
+			def translation_path
+				@translation_path ||= name&.dup.tap do |n|
+					n.gsub!("::", ".")
+					n.gsub!(/([a-z])([A-Z])/, '\1_\2')
+					n.downcase!
+				end
```

Comment:
This is the logic that already includes [the fix](https://github.com/phlex-ruby/phlex-translation/pull/1) for properly converting `Admin::Users::ShowView` into `admin.users.show_view` instead of `admin.users.showview`.